addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -297,6 +312,10 @@ export class AnalyticDistribution extends Component {
         return this.state.list;
     }
 
+    get needsPlans() {
+        return this.editingRecord && !this.allPlans.length;
+    }
```

Comment:
Why do you need this getter? It only makes you scroll more

Reply:
removed